source/openapi-admin-v3.yaml

```diff
@@ -207,8 +223,35 @@ paths:
       operationId: adminDeleteSession
       summary: Delete a session access token
       responses:
+        "204":
+          description: Deleted
         "401":
-          description: "invalid refresh token: incorrect token type"
+          description: Invalid token or session
+          content:
+            application/json:
+              schema:
+                oneOf:
+                  - allOf: 
+                    - $ref: "#/components/schemas/Error"
+                    - properties:
+                        error:
+                          const: "invalid refresh token: incorrect token type"
+                      description: |-
+                        The request did not use bearer token
+                        authentication with a valid refresh token. Make
+                        sure the Authorization header contains the
+                        `refresh_token`, not the `access_token`.
+                  - allOf: 
+                    - $ref: "#/components/schemas/Error"
+                    - properties:
+                        error:
+                          const: "invalid session: failed to find refresh token"
+                        error_code:
+                          const: "InvalidSession"
+                      description: |-
+                        The provided refresh token does not match any
+                        current sessions.
```

Comment:
might wanna make this a component. i suspect other endpoints would have this error when the errors are fully documented.